src/firebase_functions/db.py

```diff
@@ -0,0 +1,211 @@
+"""
+Module for Cloud Functions that are triggered by the Firebase Realtime Database.
+"""
+# pylint: disable=protected-access
+import dataclasses as _dataclass
```

Comment:
Please follow the Google Python style guide, including its [guidance for imports](https://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
The only reason I've not done this is that otherwise these imports pollute the exports of the module when users import it, e.g. 

Without `_`:

![image](https://user-images.githubusercontent.com/5347038/195204026-01290951-46ba-4449-8301-6a2363b0bb49.png)



With `_`: 

![image](https://user-images.githubusercontent.com/5347038/195204083-42ea9a55-ac60-4e1b-87b1-dfeba0a4dfe7.png)


This unofficial convention is followed in a few google projects including the python admin sdk; https://github.com/firebase/firebase-admin-python/blob/master/firebase_admin/db.py

If we're ok with having all imports appear in exports of the modules I can change this back?

An alternative to prevent exports pollution is tp define an `__all__` in each file which is a list of string names of things that should be exported - but this is often overlooked when updates happen since it's a manually created list of string names of module members.